CefSharp/IDomNode.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright © 2010-2015 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Represents the data belonging to a node in the browser's DOM.
```

Comment:
Probably should mention that it only represents a subset of the DOM node's information.